models/perp_schema.js

```diff
@@ -0,0 +1,43 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const PerpSchema = new Schema({
+	name: {
+		type: String,
+		required: true
+	},
+	phone: String,
+	email: String,
+	perpType: {
+		type: String,
+		required: true
+	},
+	adServiceUsed: String,
+	gender: {
+		type: String,
+		required: true,
+	},
+	age: {
+		type: String,
+		required: true
+	},
+	race: {
+		type: String,
+		required: true
+	},
+	height: {
+		type: String,
+		required: true
+	},
+	hair: {
+		type: String,
+		required: true
+	},
+	attributes: {
+		type: String,
+		required: true
+	},
+	vehicle: String
+});
+
+module.exports = PerpSchema;
```

Comment:
I can't remember, did we decide we wanted all these fields to be required? I thought we wanted them to exist but not be required?